Meadowlark-js/lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "lerna": "8.0.2",
-  "version": "0.4.1-pre.10",
+  "version": "0.5.0",
```

Comment:
Help! @bradbanister this probably isn't the right approach. At this point I can't tell what I'm supposed to do to get the version number to 0.5.0.

Reply:
I thought it was bumped on prelease/release GitHub action. You can manually bump Lerna with the command `lerna version x.y.z` to any version you want.